src/features/user/ManagePayouts/PayoutScheduleForm.tsx

```diff
@@ -0,0 +1,105 @@
+import { ReactElement } from 'react';
+import { useForm } from 'react-hook-form';
+import { Button, MenuItem, TextField } from '@mui/material';
+import ReactHookFormSelect from './ReactHookFormSelect';
+import StyledForm from '../../common/Layout/StyledForm';
+import i18n from '../../../../i18n';
+
+const { useTranslation, Trans } = i18n;
+
+const paymentFrequencies = [
+  'monthly',
+  'quarterly',
+  'semi-annual',
+  'annual',
+  'manual',
+];
+
+type FormData = {
+  payoutMinAmount: string;
+  scheduleFrequency: string;
+};
+
+const PayoutScheduleForm = (): ReactElement | null => {
+  const { t, ready } = useTranslation('managePayouts');
+  const { register, handleSubmit, errors, control } = useForm<FormData>({
+    mode: 'onBlur',
+  });
+
+  const onSubmit = (data: FormData) => {
+    console.log(data);
```

Comment:
Maybe this PR should be a _Draft_ as long as no API call are implemented to be called?

Reply:
I converted it to a _Draft_ now.